src/app/data-planes/sources.ts

```diff
@@ -34,7 +34,7 @@ export type DataPlaneCollectionSource = DataSourceResponse<DataPlaneCollection>
 
 export const sources = (api: KumaApi) => {
   return {
-    '/:mesh/dataplanes': async (params: CollectionParams & PaginationParams, source: Closeable) => {
+    '/meshes/:mesh/dataplanes': async (params: CollectionParams & PaginationParams, source: Closeable) => {
```

Comment:
Why do we need to also add this `/meshes` prefix ?

Reply:
I found that if you’re not careful, you can easily create non-unique paths so I’m prefixing every dynamic path segment with the corresponding term.

So, for example, if you define a source with path `/:mesh/:path` and one with `/:id/test`, depending on which is found first, you might get the wrong one. Prefixing the dynamic segments makes them unambiguous.

Reply:
Yeah everything should be prefixed to prevent accidental clashes (ideally with the name of the 'module'). This was the reasoning for the weird `/*/policy-types` and `/*/zones` whereever it was I did that. But unfortunately I forgot that was the reason 🤦